test/dotnet/scenarios.json

```diff
@@ -1,83 +1,50 @@
 {
-    "install_additional_dotnet": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "6.0.301",
-                "additionalVersions": "5.0,3.1.420"
-            }
-        }
-    },
-    "install_dotnet_3": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "3"
-            }
-        }
-    },
-    "install_dotnet_6_bullseye": {
-        "image": "debian:bullseye",
-        "features": {
-            "dotnet": {
-                "version": "6"
-            }
-        }
-    },
-    "install_dotnet_6_focal": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "6"
-            }
-        }
-    },
-    "install_dotnet_7_bullseye": {
-        "image": "debian:bullseye",
-        "features": {
-            "dotnet": {
-                "version": "7"
-            }
-        }
-    },
-    "install_dotnet_7_jammy": {
-        "image": "ubuntu:jammy",
+    "install_dotnet_lts": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "7"
+                "version": "lts"
             }
         }
     },
-    "install_dotnet_6_jammy": {
-        "image": "mcr.microsoft.com/devcontainers/base:jammy",
+    "install_dotnet_specific_release": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu-20.04",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "6"
+                "version": "3.1"
             }
         }
     },
-    "install_dotnet_latest": {
```

Comment:
Can we still keep the scenario which validates the `latest` option? I know that the test would fail when a new latest is available, but should be fine.

Reply:
There is the autogenerated test case which defaults to `latest` and this is verified in `test.sh` in a non-breaking way by comparing it to `latest.version` files from the AzureCDN. Will that suffice?

Reply:
That was a smart thing to do! ✨ 

Based on https://github.com/devcontainers/features/pull/628#discussion_r1276647264, I am looking forward to validate the `fetch_latest_sdk_version()` function. Hence, validating the `latest` installed version with `7` will provide us an assurance that the function works as expected.

Hence, once .NET 8 is released, I expect the test to fail.

Reply:
So I should add an assertion that the installed SDK is a flavor of 7.0, expecting the test will break in November?

Reply:
Yes, that would be great!

Reply:
Done in dc7a0beda42c078afcb078a1e57612885558d77a